### DISCUSSION TITLE
Does react-markdown support base64 encoded images ?

### DISCUSSION THREAD

**Discussion by marieflorescontact** [remarkjs/remark#1378] [Q&A]
Hi
I wanted to know if `react-markdown` support base64 encoded images ?  🙂
thanks!

**Comment by wooorm** [comment]
The answer is “Yes”. But you are probably looking for more than one word.
Base64 images are not safe. That is why they are not allowed by default.
See https://github.com/remarkjs/react-markdown#security for more info. And the rest of the docs on `urlTransform`.

**Reply by marieflorescontact** [reply]
thank you so much for this super fast answer 🙂
